spec/features/reservation_list_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe "Reservation index list", :type => :feature, :js => true do
+
+  let(:user){ FactoryGirl.create(:user) }
+
+  it "should get reservation lists from proper day" do
+    FactoryGirl.create(:reservation, from: '10', to: '11')
+    FactoryGirl.create(:reservation, from: '12', to: '13')
+    FactoryGirl.create(:reservation, from: '14', to: '15')
+    Reservation.create(reserved_at: (DateTime.now + 1), from: '10', to: '11')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.